packaging/hudi-timeline-server-bundle/run_server.sh

```diff
@@ -19,7 +19,7 @@
 
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 #Ensure we pick the right jar even for hive11 builds
-HOODIE_JAR=`ls -c $DIR/target/hudi-timeline-server-bundle-*.jar | grep -v test | head -1`
+HOODIE_JAR=`ls c $DIR/target/hudi-timeline-server-bundle*.jar | grep -v test | grep -v source | head -1`
```

Comment:
nit: `ls c` to `ls -c`

Reply:
nit: 
change to 
HOODIE_JAR=\`ls -c $DIR/target/hudi-timeline-server-bundle-*.jar | grep -v test | grep -v source | head -1\` should be ok(just need add `| grep -v source`).

Reply:
> nit: `ls c` to `ls -c`

done.

Reply:
> nit:
> change to
> HOODIE_JAR=`ls -c $DIR/target/hudi-timeline-server-bundle-*.jar | grep -v test | grep -v source | head -1` should be ok(just need add `| grep -v source`).

It's my fault! 
done.